packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx

```diff
@@ -58,86 +58,63 @@ NotEnoughGasProps) {
   }, [walletAddress]);
 
   return (
-    <Drawer
-      headerBarTitle={undefined}
-      size="full"
-      onCloseDrawer={onCloseDrawer}
-      visible={visible}
-      showHeaderBar={showHeaderBar}
-    >
-      <Drawer.Content>
-        <Box testId="not-enough-gas-bottom-sheet" sx={containerStyles}>
-          <CloudImage
-            imageUrl={
+    <Drawer size="threeQuarter" visible={visible} showHeaderBar={false}>
+      <Drawer.Content testId="not-enough-gas-bottom-sheet" sx={containerStyles}>
+        <CloudImage
```

Comment:
I think this needs some padding top and padding bottom to spread out the image from the heading + body text

Reply:
Yep, I'll fix this up and confirm with Cass